cypress/integration/e2e/sign-in-gate.spec.js

```diff
@@ -1,109 +1,156 @@
 /* eslint-disable no-undef */
 /* eslint-disable func-names */
 
-describe('Sign In Gate Tests', function() {
+describe('Sign In Gate Tests', function () {
     const setArticleCount = (n) => {
         // set article count for today to be n
-        localStorage.setItem('gu.history.dailyArticleCount', JSON.stringify({ value: [{ day: Math.floor(Date.now() / 86400000), count: n }] }));
-    }
+        localStorage.setItem(
+            'gu.history.dailyArticleCount',
+            JSON.stringify({
+                value: [
+                    {
+                        day: Math.floor(Date.now() / 86400000),
+                        count: n,
+                    },
+                ],
+            }),
+        );
+    };
 
     const setMvtCookie = (str) => {
         cy.setCookie('GU_mvt_id_local', str, {
-            log: true
-        })
-    }
+            log: true,
+        });
+    };
 
     // helper method over the cypress visit method to avoid having to repeat the same url by setting a default
     // can override the parameter if required
-    const visitArticle = (url = 'https://www.theguardian.com/games/2018/aug/23/nier-automata-yoko-taro-interview') => {
-        cy.visit(
-            `Article?url=${url}`,
-        );
-    }
+    const visitArticle = (
+        url = 'https://www.theguardian.com/games/2018/aug/23/nier-automata-yoko-taro-interview',
+    ) => {
+        cy.visit(`Article?url=${url}`);
+    };
 
     // as the sign in gate is lazy loaded, we need to scroll to the rough position where it
     // will be inserted to make it visible
     // can override position if required
     const scrollToGateForLazyLoading = (roughPosition = 1000) => {
         cy.scrollTo(0, roughPosition, { duration: 500 });
-    }
+    };
 
     // we call visit and scroll for most test, so this wrapper combines the two
     // while preserving the ability to set the parameters if required
-    const visitArticleAndScrollToGateForLazyLoad = ({ url, roughPosition } = {}) => {
+    const visitArticleAndScrollToGateForLazyLoad = ({
+        url,
+        roughPosition,
+    } = {}) => {
         visitArticle(url);
         scrollToGateForLazyLoading(roughPosition);
-    }
+    };
+
+    // prevents the CMP banner from showing which creates a race condition that prevents tests from passing
+    // use before each test
+    const setConsentCookieToBlockCMPBanner = () => {
+        cy.setCookie('consentUUID', '9c1d1bdd-3322-416a-ad9f-33773423e17c', {
```

Comment:
This is now set before every test so you probably don't need it here. See https://github.com/guardian/dotcom-rendering/pull/1829

Reply:
ok - thanks!